docs/api/request.md

```diff
@@ -117,10 +117,10 @@ this.request.charset
 
   For example "color=blue&size=small":
 
-```js
+```json
 {
-  color: 'blue',
-  size: 'small'
+  "color": "blue",
+  "size": "small"
```

Comment:
The object notation represents an expression. However, JS code consist of statements. It is possible to have *Expression Statements* (e.g. `'use strict';`) but these may not ‘start with an opening curly brace because that might make it ambiguous with a Block’, see http://es5.github.io/#x12.4. Wrapping it in parentheses would make it valid, though.